<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <widget class="Line" name="seprator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>911</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="InputLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>INPUT</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="InputEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>441</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter the text which you want to encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EncryptBtn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="OutputLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OUTPUT</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="OutputEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>631</width>
      <height>171</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>This will give the encryptd text</string>
    </property>
   </widget>
   <widget class="QComboBox" name="HashCombo">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>BLAKE2S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BLAKE2B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fernet</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="KeyEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>eneter the ket it should not be one more then a multiple of 4</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="EncryptMenu">
    <property name="title">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QMenu" name="KEncryptMenu">
    <property name="title">
     <string>Key Encrypt</string>
    </property>
   </widget>
   <widget class="QMenu" name="KDecryptMenu">
    <property name="title">
     <string>Key Decrypt</string>
    </property>
   </widget>
   <widget class="QMenu" name="PasswordMenu">
    <property name="title">
     <string>Strong password</string>
    </property>
   </widget>
   <addaction name="EncryptMenu"/>
   <addaction name="KEncryptMenu"/>
   <addaction name="KDecryptMenu"/>
   <addaction name="PasswordMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
